public/src/components/Steps.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {Grid, Row, Col} from 'react-bootstrap';
+const Steps = (props) => (
+  <Grid>
+    {props.steps.map((step, index) =>
+    {if((index + 1) % 2 !== 0) {
+      return (<Row key={index} className="show-grid">
+        <Col lg={6}>
+          {console.log('starting new row', step)}
+          <div>
+            <div>
+              <img className="image" src={step.photo} />
+            </div>
+              <div className="instructions-content">
+                <div className="step-title">
+                  <span className="step-number">{index + 1}</span>
+                  <span className="step-title-title">{step.title}</span>
+                </div>
+                <p>{step.instruction}</p>
+              </div>
+          </div>
+        </Col>
+        <Col lg={6}>
+          {console.log('continuing on same row')}
+          {/* <div>
+            <img src={props.steps[index + 1].photo} />
+            <h4>{index + 2}</h4>
+            <h4>{props.steps[index + 1].title}</h4>
+            <p>{props.steps[index + 1].instruction}</p>
+          </div> */}
+          <div>
+            <div>
+              <img className="image" src={props.steps[index + 1].photo} />
```

Comment:
This repetition can be broken  out into It's own component quite easily. It would make reading this code mush easier as well